webhook.go

```diff
@@ -194,7 +195,12 @@ func main() {
 		fmt.Fprint(w, "OK")
 	})
 
-	r.HandleFunc(hooksURL, hookHandler)
+	if *httpMethods == "" {
+		r.HandleFunc(hooksURL, hookHandler)
+	} else {
+		allowed := strings.Split(*httpMethods, ",")
```

Comment:
Could we also trim spaces from these strings after splitting them so that we don't have to explain users who like to add space after comma why it isn't working for them. :-)

Reply:
Done